lib/bulk_data_test_kit/v2.0.0_client/bulk_data_client_wait_test.rb

```diff
@@ -22,18 +21,17 @@ class ExportWaitTest < Inferno::Test
           message: %(
```

Comment:
You can add a timeout like [here in PDex](https://github.com/inferno-framework/davinci-pdex-test-kit/blob/main/lib/davinci_pdex_test_kit/pdex_payer_client/client_workflow_interaction_test.rb#L44) to increase the time Inferno will keep waiting. Seems worthwhile here.